.travis.yml

```diff
@@ -190,6 +190,9 @@ matrix:
         - os: linux
           env: SETUP_CMD='test' CONDA_DEPENDENCIES='matplotlib'
                CONDA_DEPENDENCIES_FLAGS='--no-deps'
+               NUMPY_VERSION=stable PYTHON_VERSION=3.5
+               CONDA_CHANNELS='conda-forge'
```

Comment:
why conda-forge?

Reply:
because conda-forge misses the numpy 1.16 build for py35

Reply:
but the exercise here is to check whether your patch works, isn't it?

Reply:
Right?! This is to test if the "pip fallback" works as intended for a missing numpy build. The other Travis tests also work as intended, so my PR didn't brake anything if a suitable build is available.

Reply:
Oh, sorry, I misread your comment. The default channel doesn't have 1.16 for python 3.5 either, and that preoccupied my thoughts.

Reply:
* In test 4, for example, numpy is [successfully installed via conda](https://travis-ci.org/astropy/ci-helpers/jobs/502868436#L1644) and the fallback is not triggered. So the fix works as intended.
* In test 29, [installing numpy via conda fails](https://travis-ci.org/astropy/ci-helpers/jobs/502868461#L660), [numpy is added to `PIP_DEPENDENCIES`](https://travis-ci.org/astropy/ci-helpers/jobs/502868461#L667), and is later [installed via pip](https://travis-ci.org/astropy/ci-helpers/jobs/502868461#L706).
* Do we need an additional test for an exact version specification (1.16.2)?

Reply:
>The default channel doesn't have 1.16 for python 3.5 either

Sorry I wasn't sure about this, so I used conda-forge.

Reply:
Hmm... I think the test should explicitly state the numpy version instead of using "stable" since that's subject to change... I'm pretty sure that Python 3.5 support for numpy will be dropped in ~2020 completely, which means that the pip install will also fail for a then "latest stable" numpy version.

Reply:
I pinned the version in test 29 to 1.16 and added an additional test for 1.16.1

Reply:
> I'm pretty sure that Python 3.5 support for numpy will be dropped in ~2020 completely

you're very optimistic that we still have travis in 2020 :)